app/controllers/application_controller.rb

```diff
@@ -1,10 +1,20 @@
 require Rails.root.join("commands", "authorize_api_request")
+require Rails.root.join("commands", "authenticate_user")
 
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :null_session
 
   before_action :authenticate_request
 
+  def authenticate
```

Comment:
Maybe you could make the method take 2 parameters for email and password instead of using the http params.  I was confused about how it was working at first.  You'd have to change the route for login